client/src/templates/Challenges/redux/index.js

```diff
@@ -142,6 +143,7 @@ export const unlockCode = createAction(types.unlockCode);
 export const disableBuildOnError = createAction(types.disableBuildOnError);
 export const storedCodeFound = createAction(types.storedCodeFound);
 export const noStoredCodeFound = createAction(types.noStoredCodeFound);
+export const saveCode = createAction(types.saveCode);
```

Comment:
I think `saveEditorContents` is more descriptive. `code` can mean many things given a different context.